libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -16,12 +17,12 @@
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
     <div class="book" data-testing="book-item" *ngFor="let b of books">
-      <div class="book--title">
+      <div class="book--title" aria-label="bookTitle">
         {{ b.title }}
       </div>
       <div class="book--content">
         <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+          <img src="{{ b.coverUrl }}" alt="Book Content Cover"/>
```

Comment:
This might get a bit repetitive for screen readers.

Reply:
I see what you mean. I should make it specific to the book